Makefile

```diff
@@ -3,14 +3,10 @@ all: form.pdf
 %.pdf: %.svg
 	inkscape -A $@ $<
 
-marketing/logos:
```

Comment:
Why are we _removing_ this target?
I appreciate that the submodule has changed, however I still think it's useful to have a target of this form.